### ISSUE TITLE
Req: Attachment Extraction using dynamic paths

### ISSUE THREAD

**Issue by MichiFr** [RealRaven2000/quickFilters#216]
Hi Axel!

Maybe a feature request, maybe already possible beyond my knowledge ;-) 
quickFilters and/or FiltaQuilla both have one or two filter actions to extract attachments into a user defined folder. 

Is it possible to have some placeholders in the folder path to get dynamic paths? Like H:\attachments\%**year**%\ or H:\attachments\%**year**%\%**month**%\ and similar date variables?

Thanks,
Michael

**Comment by RealRaven2000** [comment]
> Hi Alex!
> 
> Maybe a feature request, maybe already possible beyond my knowledge ;-) quickFilters and/or FiltaQuilla both have one or two filter actions to extract attachments into a user defined folder.
> 
> Is it possible to have some placeholders in the folder path to get dynamic paths? Like H:\attachments%**year**%\ or H:\attachments%**year**%%**month**%\ and similar date variables?

no, anything with dynamic folder names is difficult to do because it requires asynchronous actions. Filter actions only support synchronous functions, by design.

**Comment by MichiFr** [comment]
Ok, understood and makes sense of course, assuming that the rule creates the appropriate (dynamic) folder, correct?

However, I do not want the folder to be **created** automatically. The folder already exists, created manually. My intention is to not modify the filter each year or month and insert the current date.

Similar using _custom variables_ in this picture from your website.
![image](https://github.com/RealRaven2000/quickFilters/assets/28045390/8cbe4ec4-7c7d-4cbc-ac6c-d72dac49fd1c)
